src/dashboard/trade/placeOrder/AmountDetails.tsx

```diff
@@ -21,12 +21,72 @@ type AmountDetailsProps = {
 const useStyles = makeStyles(() => {
   return createStyles({
     iconContainer: {
+      color: "#979797",
       display: "flex",
       alignItems: "center",
+      fontSize: "16px",
+    },
+    text: {
+      color: "#979797",
+      fontSize: "16px",
+      padding: "2px 0",
+    },
+    textHighlighted: {
+      color: "#f2f2f2",
+    },
+    toolTip: {
```

Comment:
I suppose all the tooltips should look alike, so why not create a component that has those styles and can be used anywhere?